packages/api/src/routers/blob/getByVersionedHash.ts

```diff
@@ -3,12 +3,24 @@ import { TRPCError } from "@trpc/server";
 import type { BlobReference } from "@blobscan/blob-storage-manager";
 
 import { publicProcedure } from "../../procedures";
-import { getByVersionedHashInputSchema } from "./getByVersionedHash.schema";
+import {
+  getByVersionedHashInputSchema,
+  getByVersionedHashOutputSchema,
+} from "./getByVersionedHash.schema";
 
 export const getByVersionedHash = publicProcedure
```

Comment:
Maybe we can create an unified procedure called `getByBlobId` similar to the block id I mentioned below.

The blob id could be the versioned hash or the commitment.

Wdyt?